templates/_helpers.tpl

```diff
@@ -190,7 +197,7 @@ Get the report db URL
         {{ .Values.external.db.report.url }}
     {{- else -}}
         {{- if .Values.postgresql.install -}}
-            jdbc:postgresql://{{ include "postgresql.subchart" . }}:{{ .Values.postgresql.service.port }}/xlr-report-db
+            jdbc:postgresql://{{ include "postgresql.subchart" . }}:{{ .Values.postgresql.primary.service.ports.postgresql }}/xlr-report-db
```

Comment:
This is not valid change, the state of the branch is not correct

Reply:
Merged all changes from the maintenance branch.